components/frontend/src/App.js

```diff
@@ -218,7 +235,7 @@ class App extends Component {
                 handleDateChange={(e, { name, value }) => this.handleDateChange(e, { name, value })}
                 last_update={this.state.last_update}
                 loading={this.state.loading}
-                nr_measurements={this.state.nr_measurements}
+                nr_measurements={this.state.nrMeasurements}
```

Comment:
Why keep `nr_measurements`, is it a large refactor?

Reply:
The refactoring is pretty small, but if the code touched doesn't have tests SonarQube is going to punish me for it :-)